src/zkevm_specs/evm/instruction.py

```diff
@@ -368,6 +377,42 @@ def range_check(self, value: Expression, n_bytes: int) -> bytes:
         except OverflowError:
             raise ConstraintUnsatFailure(f"Value {value} has too many bytes to fit {n_bytes} bytes")
 
+    def constrain_abs_word(self, x: RLC, x_abs: RLC, x_is_neg: FQ):
```

Comment:
no need for this function.

Reply:
Deleted this function `constrain_abs_word `.